SR2020/2019-10-21-kit-disclaimer-form.md

```diff
@@ -0,0 +1,22 @@
+---
+to: sr2020-teams
+subject: Kit Disclaimer Forms
```

Comment:
Sorry, should have suggested in Slack -- since this is now about more than just kit forms, maybe update the subject line?

I can't actually think of anything good right now, though here are some thoughts:
- Kit distribution & Kickstart details; implies there's more kickstart details, which there kinda aren't
- Looking forward to Kickstart (or similar); could easily be ignored by non-kickstart attending teams

Reply:
I've gone with the former, as there are still _some_ kickstart details in the email.